tests/e2e/test_rosacli_ingress.go

```diff
@@ -178,6 +179,41 @@ var _ = Describe("Edit default ingress",
 				ingress, _ = defaultIngress(*ingressList)
 				Expect(ingress.LBType).Should(ContainSubstring(ingress.LBType))
 			})
+
+		It("can customize ingress controller at install - [id:65798]",
+			labels.High,
+			labels.Day1Post,
```

Comment:
[Here](https://github.com/openshift/rosa/pull/2093/files#diff-b1d5ebe3991fe8d02791246cbf7cdc7a8eabf417254dd7c3e47c4ae6701acc19R17) defines the cases under this group is `Day2`, is this `Day1Post`? We need to remove the `Day2` from `Description` but add it to each case.